src/main/java/com/sap/oss/phosphor/fosstars/data/github/UnpatchedVulnerabilities.java

```diff
@@ -89,6 +89,7 @@ public UnpatchedVulnerabilities(GitHubDataFetcher fetcher, NVD nvd) throws IOExc
   Vulnerabilities vulnerabilitiesFromNvdFor(GitHubProject project) throws IOException {
     Vulnerabilities vulnerabilities = new Vulnerabilities();
     for (NvdEntry entry : nvd.search(NvdEntryMatcher.entriesFor(project))) {
+      logger.info("Found entry {}", entry.getCve());
```

Comment:
Do we really need it?

```suggestion
      logger.debug("Found entry {}", entry.getCve());
```